pages/common/plesk.md

```diff
@@ -0,0 +1,28 @@
+# plesk
+
+> Plesk hosting control panel CLI interface.
+> More information: <https://docs.plesk.com>.
+
+- Generate an auto login link for the admin user:
+
+`plesk login`
+
+- Show product version information:
+
+`plesk version`
+
+- List all hosted domains:
+
+`plesk bin domain --list`
+
+- Start watching for changes in the `panel.log` file:
+
+`plesk log panel.log`
+
+- Start the interactive MySQL console:
+
+`plesk db`
+
+- Open the Plesk main configuration file in the default editor:
+
+`plesk conf panel.ini`
```

Comment:
Does this command assume `panel.ini` in the in the CWD?  Or does `plesk` find the file somewhere when it's given just  a file name?

Reply:
No, `panel.ini` is located inside Plesk installation directory, not in CWD. The `plesk` utility knows how to find the appropriate config by given name.

Reply:
Does [my question](https://github.com/tldr-pages/tldr/pull/5698#discussion_r610573505) apply here, with this subcommand?

Reply:
The same problem with the example and description is here (as described above). There are two options: panel.ini and php.ini. But panel.ini option is used in 99% of cases. So let's keep the original example.

Reply:
Great.